chromebook-setup.sh

```diff
@@ -650,6 +694,99 @@ cmd_eject_storage()
     echo "All done."
 }
 
+cmd_setup_fedora_rootfs()
```

Comment:
Can we add a comment like this just before these functions?

+# -----------------------------------------------------------------------------
+# Experimental: Create Fedora images for Chromebooks